package protoFile.afk.LevelUpSpeedWaJue2803;
message LevelUpSpeedWaJueRequest{
    required int32 id=1;//角色id
    required int32 type=2;//加速类型
}
message LevelUpSpeedWaJueResponse{
    optional string message=1;
    optional bool result=2; 
}